content/docs/examples/tutorial/setup-kubernetes-cluster/index.md

```diff
@@ -0,0 +1,208 @@
+    tutorial with your friends using your cluster), you may want to allocate a separate namespace per each
+    participant. The tutorial supports work in multiple namespaces simultaneously by multiple participants.
+    {{< /tip >}}
+
+1.  Install Istio with strict mutual TLS enabled. The simplest way is to follow
+    [these instructions](/docs/setup/kubernetes/install/kubernetes/#installation-steps),
+    select the `strict mutual TLS` tab.
+
+1.  [Enable Envoy's access logging](/docs/tasks/telemetry/logs/access-log/#enable-envoy-s-access-logging).
+
+1.  Create a Kubernetes Ingress resource for common Istio services:
+
+    - [Grafana](https://grafana.com/docs/guides/getting_started/)
+    - [Jaeger](https://www.jaegertracing.io/docs/1.13/getting-started/)
+    - [Prometheus](https://prometheus.io/docs/prometheus/latest/getting_started/)
+    - [Kiali](https://www.kiali.io/documentation/getting-started/)
+
+    If you do not know what are the services above - great, this tutorial will teach you about each of them.
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: extensions/v1beta1
+    kind: Ingress
+    metadata:
+      name: istio-system
+      namespace: istio-system
+    spec:
+      rules:
+      - host: my-istio-dashboard.io
+        http:
+          paths:
+          - path: /
+            backend:
+              serviceName: grafana
+              servicePort: 3000
+      - host: my-istio-tracing.io
+        http:
+          paths:
+          - path: /
+            backend:
+              serviceName: tracing
+              servicePort: 80
+      - host: my-istio-logs-database.io
+        http:
+          paths:
+          - path: /
+            backend:
+              serviceName: prometheus
+              servicePort: 9090
+      - host: my-kiali.io
+        http:
+          paths:
+          - path: /
+            backend:
+              serviceName: kiali
+              servicePort: 20001
+    EOF
+    {{< /text >}}
+
+1.  You may want to limit the permissions of each participant so they will be able to create resources only in their
+    namespace. Even if you run the tutorial alone on your own cluster, it could be a good practice to limit yourself so
+    your learning will not interfere with other namespaces in your cluster.
+
+    Perform the following steps to achieve this:
+
+    1.  Create a role to provide read access to `istio-system` namespace:
+
+        {{< text bash >}}
+        $ kubectl apply -f - <<EOF
+        kind: Role
+        apiVersion: rbac.authorization.k8s.io/v1beta1
+        metadata:
+          name: istio-system-access
+          namespace: istio-system
+        rules:
+        - apiGroups: ["", "extensions", "apps"]
+          resources: ["*"]
+          verbs: ["get", "list"]
+        EOF
+        {{< /text >}}
+
+    1.  Create a service account for the user of the namespace and provide access to the namespace's resources to that
```

Comment:
@rcaballeromx Yes, I did this opaque since it is not part of the tutorial, it is instructions to setup a cluster. I had some people working with these instructions to setup a cluster for a workshop. They created namespaces and limited permissions for 17 participants and they had not questions about this specific part. Let's proceed and get feedback from the users if they need more explanations.